docfx_yaml/extension.py

```diff
@@ -1482,12 +1511,22 @@ def find_markdown_pages(app, outdir):
 
                     prepend_markdown_header(name, mdfile_iterator)
 
-            shutil.copy(mdfile, f"{outdir}/{mdfile.name.lower()}")
+            mdfile_name_to_use = mdfile.name.lower()
+            if mdfile_name_to_use in files_to_rename:
+                mdfile_name_to_use = files_to_rename[mdfile_name_to_use]
+
+            shutil.copy(mdfile, f"{outdir}/{mdfile_name_to_use}")
+
+            # Do not add index file to the TOC.
```

Comment:
Why not? Seems like we could set `name` to `Overview` here and not need the special handling down below (`[{'name': 'Overview', 'href': 'index.md'}] + app.env.markdown_pages + pkg_toc_yaml`)?

Perhaps for sorting? Still seems like it would be better to handle all md pages in this function.